### ISSUE TITLE
What's the difference between acc and val_acc

### ISSUE THREAD

**Issue by y-rok** [tflearn/tflearn#357]
When I run the tfLearn example, i get logs printing acc and val_acc
As i understand, val_acc is the accuracy of validation set
However, I am not sure what acc identifies exactly.
Does "acc" mean the accuracy of all of training data of some parts of training data?

And what is validation_batch_size? 
Im confused why batch size is needed...( I know what batch size of training data means)

Thanks in advance

**Comment by aymericdamien** [comment]
val_acc is the accuracy computed on the validation set (data that have never been 'seen' by the model). batch size for testing is exactly the same concept as training batch size, you usually cannot load all your testing data into memorym so you ahve to use batches. If you do not specify a batch, the training batch size will be used for testing too.

**Comment by jyoungrok** [comment]
@aymericdamien  Thanks for your comment
In summary, acc is the accuracy of a batch of training data and val_acc is the accuracy of a batch of testing data.
Is it correct?

**Comment by aymericdamien** [comment]
yes

**Comment by TrinhTUHH** [comment]
Why do we need accuracy of training data which is the data that model have already seen?

**Comment by hcchengithub** [comment]
@TrinhTUHH 'already seen' != 'already remembered', a neural network does not remember the data, if it did that'd be an overfitting, so the same data's accuracy is not 100%. I think so, am new too.

**Comment by morenoh149** [comment]
@y-rok if your issue is resolved please close

**Comment by y-rok** [comment]
@morenoh149 Sorry...i forgot to close this issue.
